image-classifier/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "image-classifier",
+  "version": "1.0.0",
+  "description": "Image Classifier demo",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --output-pathinfo --progress",
+    "build": "npx webpack --env NODE_ENV=production"
+  },
+  "devDependencies": {
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "html-webpack-plugin": "^5.5.0",
+    "ts-loader": "^9.4.2",
+    "typescript": "^4.9.4",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-server": "^4.11.1"
+  },
+  "dependencies": {
+    "@mediapipe/tasks-vision": "0.1.0-alpha-1"
```

Comment:
Should we use `@latest` here or will we have a pipeline that updates the versions? (granted, `@latest` won't work if we check in the package-lock.json)

Reply:
@schmidt-sebastian cc @khanhlvg 
> @latest won't work if we check in the package-lock.json
That right! 

About using `@latest` and removing `package-lock.json`, we can install newest tasks-vision version in the future. However, the current sample code may not compatible with newest version.

In the other hand, keep exactly help sample code runnable with less effect from tasks-vision update. For sure, we need the pipeline to update the versions.

Reply:
Discussed internally - we'll hard-code latest version number in doc and bump the version number whenever there's new release. We can avoid API breaking changes to break code snippet in documentation.